GIFrameworkMaps.Web/Views/Shared/_MapLayout.cshtml

```diff
@@ -65,7 +65,12 @@
                         }
                         
                     </ul>
-                    <partial name="_LoginPartial" />
+                    @{
+                        if (Model.ShowLogin == true)
```

Comment:
This is using a dynamic model property which, whilst it does work, is not ideal as it could cause unexpected issues.

This also doesn't quite follow the brief. The login partial view should still show if a user is logged in, even if ShowLogin has been set to false. As it stands, this will hide the account box in the top right even if the user is logged in.